ballerina/tests/test.bal

```diff
@@ -0,0 +1,249 @@
+import ballerina/io;
+import ballerina/test;
+
+configurable boolean isLiveServer = true;
+configurable string serviceUrl = isLiveServer ? "https://api.hubapi.com" : "http://localhost:9090";
+configurable boolean isOauth=?;
+configurable string oauthKey=?;
+configurable string apiKey=?;
+int:Signed32 appId=5712614;
+
+ // BearerTokenConfig
+ConnectionConfig oauthConfig = {
+    auth: {
+        token:oauthKey
+        }
+};
+
+  
+ // API Key Config
+ConnectionConfig apikeyConfig = {auth:{
+
+    hapikey:"42c4fd68-72d9-49e7-8254-520f9acce400" ,
```

Comment:
Is this a valid access token, if so lets not commit this.

Reply:
ok